src/Status.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace Yiisoft\Http;
+
+/**
+ * HTTP status codes
+ */
+final class Status
+{
+    public const CONTINUE = 100;
+    public const SWITCHING_PROTOCOLS = 101;
+    public const PROCESSING = 102;
+    public const CONNECTION_TIMEOUT = 118;
+    public const OK = 200;
+    public const CREATED = 201;
+    public const ACCEPTED = 202;
+    public const NON_AUTHORITATIVE = 203;
+    public const NO_CONTENT = 204;
+    public const RESET_CONTENT = 205;
+    public const PARTIAL_CONTENT = 206;
+    public const MULTI_STATUS = 207;
+    public const ALREADY_REPORTED = 208;
+    public const CONTENT_DIFFERENT = 210;
+    public const IM_USED = 226;
+    public const MULTIPLE_CHOICES = 300;
+    public const MOVED_PERMANENTLY = 301;
+    public const FOUND = 302;
+    public const SEE_OTHER = 303;
+    public const NOT_MODIFIED = 304;
+    public const USE_PROXY = 305;
+    public const RESERVED = 306;
+    public const TEMPORARY_REDIRECT = 307;
+    public const PERMANENT_REDIRECT = 308;
+    public const TOO_MANY_REDIRECT = 310;
+    public const BAD_REQUEST = 400;
+    public const UNAUTHORIZED = 401;
+    public const PAYMENT_REQUIRED = 402;
+    public const FORBIDDEN = 403;
+    public const NOT_FOUND = 404;
+    public const METHOD_NOT_ALLOWED = 405;
+    public const NOT_ACCEPTABLE = 406;
+    public const PROXY_AUTHENTICATION_REQUIRED = 407;
+    public const REQUEST_TIMEOUT = 408;
+    public const CONFLICT = 409;
+    public const GONE = 410;
+    public const LENGTH_REQUIRED = 411;
+    public const PRECONDITION_FAILED = 412;
+    public const REQUEST_ENTITY_TOO_LARGE = 413;
```

Comment:
Isn't thins one called "Payload Too Large" in the spec? https://tools.ietf.org/html/rfc7231#section-6.5.11